documents/write_acls.go

```diff
@@ -0,0 +1,86 @@
+package documents
+
+import (
+	"bytes"
+
+	"github.com/centrifuge/precise-proofs/proofs"
+)
+
+// changedField holds the compact property, old and new value of the field that is changed
+// if the old is nil, then it is a set operation
+// if new is nil, then it is an unset operation
+// if both old and new are set, then it is an edit operation
+type changedField struct {
+	property, old, new []byte
```

Comment:
Do you think it might be interesting to store the readable name as well to be returned?

Reply:
TBH, i'm not sure at the moment, if we need it somewhere, we can add it here then 👍